flow/scheduling/base.py

```diff
@@ -209,9 +209,7 @@ def _call_submit(submit_cmd, script, pretend):
             tmp_submit_script.flush()
             submit_cmd.append(tmp_submit_script.name)
             try:
-                subprocess.check_output(
-                    submit_cmd, stderr=subprocess.STDOUT, universal_newlines=True
-                )
+                subprocess.check_output(submit_cmd, stderr=subprocess.STDOUT, text=True)
```

Comment:
This change is part of the update to `pyupgrade`. This update is possible because we're using the flag `--py38-plus`. It's nice to have tools to automate these kinds of updates to Python syntax. :)

Reply:
wow that is amazing!